site/docs/4.6.0/overview/releaseNotes.md

```diff
@@ -2,16 +2,61 @@
 title: Apache BookKeeper 4.6.0 Release Notes
 ---
 
-[provide a summary of this release]
+This is the seventh release of BookKeeper as an Apache Top Level Project!
+
+The 4.6.0 release incorporates new fixes, improvements, and features since previous major release 4.5.0.
 
 Apache BookKeeper users are encouraged to upgrade to 4.6.0. The technical details of this release are summarized
 below.
 
 ## Highlights
 
-[List the highlights]
+The main features in 4.6.0 cover are around following areas:
+- Dependencies Upgrade
+- Bookie enhancement
+- BookKeeper Admin REST API
+- New BookKeeper API
+- Performance improvement
+- Kubernetes deployment for BookKeeper Docker 
+
+### Dependencies Upgrade
+
+- Upgrade Protobuf to `3.4`.
+
+### Bookie enhancement
+
+- Persistable bookie status.
+  - Prior to this release, bookie status was transient. It is a bit hard for management tooling. This feature adds persistable bookies status. See [Issue-265](https://github.com/apache/bookkeeper/issues/265) for more details.
+
+- Introduce Bookie Registration Manager. 
```

Comment:
"Introduce Bookie Discovery Interface" or "Introduce Bookie Registration Interface"

"Prior to this release, bookkeeper client only provides interfaces for ledger metadata management. It doesn't provide any interface for service discovery part. This feature introduces bookie discovery interface, so it allows plugging in different service discovery backends for bookkeeper."

Reply:
Thanks for the instructions. will change it.